app/components/check_answers_summary_list_card_component.rb

```diff
@@ -1,10 +1,12 @@
 class CheckAnswersSummaryListCardComponent < ViewComponent::Base
-  attr_reader :questions, :log, :user
+  attr_reader :questions, :log, :user, :bulk_upload
 
-  def initialize(questions:, log:, user:)
+  def initialize(questions:, log:, user:, bulk_upload: nil)
     @questions = questions
     @log = log
     @user = user
+    @bulk_upload = bulk_upload
```

Comment:
Not against doing it like this, but checking if @log has bulk_upload_id might mean that we pass in less arguments here

Reply:
The behaviour would be different if it were implemented that way. Here the red version is only shown when the user is filtering logs via bulk upload. In your implementation it would always display the red version if a log was uploaded via bulk upload regardless if they were filtering by bulk upload. I will check in with designers to see which it is they are after

Reply:
I've tweaked this now, so it's always red if the log was bulk uploaded and ignores any filtering the user has chosen